packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1004,12 +993,8 @@ export interface CallWithChatAdapterManagement {
     disposeRemoteVideoStreamView(remoteUserId: string): Promise<void>;
     disposeScreenShareStreamView(remoteUserId: string): Promise<void>;
     disposeStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void>;
-    // (undocumented)
-    downloadAttachment: (options: {
-        attachmentUrl: string;
-    }) => Promise<AttachmentDownloadResult>;
-    // (undocumented)
-    downloadResourceToCache(resourceDetails: ResourceDetails): void;
+    // @beta (undocumented)
```

Comment:
Do we care about these (undocumented) tags?

Reply:
> Do we care about these (undocumented) tags?

I've asked the same question in Josh's PR. This function is actually documented but the api is not generated correctly.